src/pudl/metadata/classes.py

```diff
@@ -901,7 +907,7 @@ class DataSource(Base):
 
     name: SnakeCase
     title: String = None
-    description: String = None
+    description: Dict[SnakeCase, Any] = {}
```

Comment:
Why does description become a dictionary?

Reply:
Are there any other places where this structure was being used that might break now? Where were we outputting `datapackage.json` files based on these fields? I can't remember if we got as far as relying on this information in the `pudl-zenodo-storage` repo.